db/tasks.php

```diff
@@ -89,5 +89,14 @@
         'dayofweek' => '*',
         'month'     => '*'
     ),
+    [
+        'classname' => 'tool_objectfs\task\sync_filedir_location',
+        'blocking'  => 0,
+        'minute'    => '0',
+        'hour'      => '*/8',
```

Comment:
Can you run this just once a night, say 2am

Reply:
updated